drivers/volume/portworx/portworx.go

```diff
@@ -1117,7 +1118,7 @@ func parseGroupLabelsFromAnnotations(annotations map[string]string) map[string]s
 			continue
 		}
 
-		matches := pxAnnotationKeyRegex.FindStringSubmatch(k)
```

Comment:
The changes to filter out labels above isn't really required.

Reply:
Kept pxSnapshotGroupIDKey and removed others. Filtering out pxSnapshotGroupIDKey is required since that's a special annotation for group ID and not for generic selector labels.